tests.py

```diff
@@ -163,6 +163,26 @@ def gen():
     assert list(gen()) == [1, 2, 3]
 
 
+def test_exception_before_first_yield():
+    """Test an exception raised before the first `yield`."""
+
+    def raise_value_error():
+        raise ValueError()
+
+    @yieldfrom
+    def subgen():
+        yield raise_value_error()
```

Comment:
In our case, instead of this line we had: 
`yield From(raise_value_error())`

Is that affects the overall behaviour?

Reply:
The `From` is just a red herring in this case, it's never executed.